src/index.html

```diff
@@ -11,8 +14,71 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page__body">
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product-image">
+        <img
+          class="product-image"
+          src="images/imac.jpeg"
+          alt="product image"
+        />
+      </div>
+
+      <div class="card__product-name">
+        <p>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      </div>
+
+      <div class="card__product-code">
+        <p>Product code: 195434</p>
+      </div>
+
+      <div class="card__review">
+        <div class="stars stars--3">
```

Comment:
```suggestion
        <div class="stars stars--4">
```
tests expected 4 stars